src/main/java/Ui.java

```diff
@@ -0,0 +1,126 @@
+import java.util.Scanner;
+
+/**
+ * Encapsulates the (command line) user interface provided to the user.
+ * The `Ui` class provides utility methods to show various message types.
+ * Messages will be displayed in standard output.
+ *
+ * <p>Most of the methods do not print any header line, with the exception of `greet()`
+ */
+public class Ui {
```

Comment:
In general, I think your Ui class is very well done which makes your code very easy to read because all the printing is done with clearly defined methods in your Ui class. Good job!